### DISCUSSION TITLE
Can't add new components

### DISCUSSION THREAD

**Discussion by FalseFeint** [shadcn-ui/ui#4991] [Q&A]
Hi I'm trying to run the following:

``` npx shadcn@latest add calendar ```

but I keep getting the following error:

```
Something went wrong. Please check the error below for more details.
If the problem persists, please open an issue on GitHub.

request to https://ui.shadcn.com/r/index.json failed, reason:
```

Any ideas what's going on? I'm able to visit https://ui.shadcn.com/r/index.json. When I try to ping I get the following:

```
$ ping ui.shadcn.com

Pinging cname.vercel-dns.com [76.76.21.142] with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.

Ping statistics for 76.76.21.142:
    Packets: Sent = 4, Received = 0, Lost = 4 (100% loss),

```

Thanks

**Comment by imyuanli** [comment]
+1，I hope to increase the number of agents, otherwise it would be too unfriendly to foreign users. Thank you

**Comment by Zitu-hoq** [comment]
it happens because of proxy.
Possible solution: 1. restart your internet connection(turn off then turn on) and reopen the ide. 
2. Use a proxy server or vpn

**Comment by imyuanli** [comment]
这是自动回复的邮件。您好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
